da/lazyledger/lazyledger.go

```diff
@@ -55,15 +70,18 @@ func (ll *LazyLedger) Init(config []byte, logger log.Logger) error {
 	if err != nil {
 		return err
 	}
+
+	ll.encCfg = RegisterAccountInterface(params.MakeEncodingConfig())
+
 	var userInput io.Reader
 	// TODO(tzdybal): this means interactive reading from stdin - shouldn't we replace this somehow?
 	userInput = os.Stdin
-	ll.keyring, err = keyring.New(ll.config.AppName, ll.config.Backend, ll.config.RootDir, userInput)
+	ll.keyring, err = keyring.New(ll.config.KeyringAccName, ll.config.Backend, ll.config.RootDir, userInput)
```

Comment:
I thought it was a tad more precise of what it was used for. It's a name in the keyring representing an account. We could use AppName, but imho that sounds like it should be the name of the app being used, not which key should be used to pay for message.

will revert if you still think otherwise :slightly_smiling_face: